contracts/sol6/Dao/KyberStaking.sol

```diff
@@ -303,6 +338,21 @@ contract KyberStaking is IKyberStaking, EpochUtils, ReentrancyGuard {
         return stakerLatestData[staker].stake;
     }
 
+    function getLatestStakerData(address staker)
+        external view
+        returns (
+            uint256 stake,
+            uint256 delegatedStake,
+            address delegatedAddress
+        )
+    {
+        stake = stakerLatestData[staker].stake;
+        delegatedStake = stakerLatestData[staker].delegatedStake;
+        delegatedAddress = stakerLatestData[staker].delegatedAddress == address(0)
+                ? staker
+                : stakerLatestData[staker].delegatedAddress;
+    }
+
     // prettier-ignore
```

Comment:
BTW
Do we still need prettier ignore here?

Some of those were left when we used shorter lines (80)

Reply:
Removed